social/templates/social/friendsearch_return.html

```diff
@@ -0,0 +1,12 @@
+{% for result in search_results %}
+    <p>
+        <a href="{% url 'viewProfile' result.id %}">{{ result.first_name }} {{ result.last_name }} | {{ result.email }}</a> 
+        <form action="{% url 'addFriend' %}" method="POST">
+            {% csrf_token %}
+            <input type="hidden" name="friendID" value="{{ result.id }}">
+            <input type="submit" value="Add {{ result.first_name }} as a friend!">
```

Comment:
can we Throw an if statement in here that says if this user is already your friend dont show the add friend? just for that extra touch?

Reply:
disregard I see the PR description!